ast/src/statements/assign/mod.rs

```diff
@@ -24,22 +24,38 @@ use std::fmt;
 mod assignee;
 pub use assignee::*;
 
+/// The assignment operator.
 #[derive(Clone, Copy, PartialEq, Eq, Serialize, Deserialize, Debug)]
 pub enum AssignOperation {
+    /// Plain assignment, `=`.
     Assign,
+    /// Add-assignment, `+=`.
```

Comment:
Say 'adding assignment' for consistency with 'subtracting assignment' and others?